Completion/Unix/Command/_code

```diff
@@ -0,0 +1,36 @@
+#compdef code
+
+local arguments
+
+arguments=(
+  '(-d --diff)'{-d,--diff}'[compare two files with each other]:file to compare:_files:file to compare with:_files'
+  '(-a --add)'\*{-a,--add}'[add specified directory (or directories) to the last active window.]:directory:_directories'
+  '(-g --goto)'{-g,--goto}'[open a file at the path on the specified line and character position]:file\:line[\:character]:_files -r \:'
+  '(-n --new-window -r --reuse-window)'{-n,--new-window}'[open a new window]'
+  '(-n --new-window -r --reuse-window)'{-r,--reuse-window}'[open a file or directory in the last active window]'
+  '(-w --wait)'{-w,--wait}'[wait for the files to be closed before returning]'
+  '--locale=[specify the locale to use]:locale (e.g. en-US or zh-TW):(de en en-US es fr it ja ko ru zh-CN zh-TW bg hu pt-br tr)'
+  '--user-data-dir[specify the directory that user data is in]:directory:_directories'
+  '(-v --version)'{-v,--version}'[print version]'
+  '(-h --help)'{-h,--help}'[print usage]'
```

Comment:
I can't verify with code at the moment but it is common for options like -h and -V to not be applicable in combination with any other options. In which case the exclusion list can be just (- *)

Reply:
Set exclusion list to `(- *)`

Maybe the obverse should be done too:
If any parameters have been added, disallow `-v` and `-h`?